apps/backoffice-v2/src/lib/blocks/variants/DefaultBlocks/hooks/useCaseBlocksLogic/hooks/useCaseTabsLogic/useCaseTabsLogic.ts

```diff
@@ -0,0 +1,31 @@
+import { TCaseTabDefinition } from '@/lib/blocks/variants/DefaultBlocks/types/case-tab';
+import { useCallback, useMemo, useState } from 'react';
+
+export const useCaseTabsLogic = (tabs: TCaseTabDefinition[]) => {
+  const [activeTabName, setActiveTabName] = useState<string | null>(() => {
+    return tabs.length ? (tabs.at(0)?.name as string) : null;
+  });
```

Comment:
Using `undefined` over `null` is simpler and the type is inferred too.
```suggestion
  const [activeTabName, setActiveTabName] = useState(() => {
    return tabs?.at(0)?.name;
  });
```